# Урок 4. Непрерывные случайные величины. Функция распределения и плотность распределения вероятностей. Равномерное и нормальное распределение. Центральная предельная теорема
1.Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800].
Найдите ее среднее значение и дисперсию.

In [1]:
def mean_and_variance(a, b):
  return f'M(A) = {(a+b)/2:.2f};\n D(A) = {((b-a)**2)/12:.2f}'
print(mean_and_variance(200, 800))

M(A) = 500.00;
 D(A) = 30000.00


2.О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2.
Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5?
Если да, найдите ее.

In [2]:
b = 0.5 + 2.4 **(1/2)
print(f'B - b = {b:.3f}')
print(f'M(B) = {(b+0.5)/2:.2f}')

B - b = 2.049
M(B) = 1.27


3.Непрерывная случайная величина X распределена нормально и задана плотностью распределения
f(x) = (1 / (4 * sqrt(2*pi))) * (exp(-(x+2)**2) / 32).
Найдите:
а). M(X)
б). D(X)
в). std(X) (среднее квадратичное отклонение)

M(X) = -2

D(X) = 16

std(X) = 4

4.Рост взрослого населения города X имеет нормальное распределение.
Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см.
Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
а). больше 182 см
б). больше 190 см
в). от 166 см до 190 см
г). от 166 см до 182 см
д). от 158 см до 190 см
е). не выше 150 см или не ниже 190 см
ё). не выше 150 см или не ниже 198 см
ж). ниже 166 см.

In [3]:
def z_value(height):
  return (height - 174)/8

In [4]:
from statistics import NormalDist

In [5]:
print(z_value(182))

1.0


In [8]:
NormalDist().cdf(z_value(182))

0.8413447460685429

In [9]:
P = 1 - NormalDist().cdf(z_value(182))
P

0.15865525393145707

In [10]:
Z = z_value(190)
Z

2.0

In [13]:
P = 1 - NormalDist().cdf(Z)
P

0.02275013194817921

In [14]:
Z1 = z_value(166)
Z1

-1.0

In [17]:
Z2 = z_value(182)
Z2

1.0

In [18]:
P = NormalDist().cdf(Z2) - NormalDist().cdf(Z1)
P

0.6826894921370859

In [19]:
Z1 = z_value(158)
Z1

-2.0

In [20]:
Z2 = z_value(190)
Z2

2.0

In [21]:
P = NormalDist().cdf(Z2) - NormalDist().cdf(Z1)
P

0.9544997361036416

In [22]:
Z1= z_value(150)
Z1

-3.0

In [23]:
Z2 = z_value(190)
Z2

2.0

In [25]:
P = NormalDist().cdf(Z1)+(1 - NormalDist().cdf(Z2))
P

0.024100029979809312

In [26]:
Z1= z_value(150)
Z1

-3.0

In [27]:
Z2= z_value(198)
Z2

3.0

In [28]:
P = NormalDist().cdf(Z1)+(1 - NormalDist().cdf(Z2))
P

0.002699796063260207

In [29]:
Z = z_value(166)
Z

-1.0

In [31]:
P = NormalDist().cdf(Z)
P

0.15865525393145707

P(X>=182) = 0.1587;

P(X>=190) = 0.0227;

P(166<=X<=190) = 0.8186;

P(166<=X<=182) = 0.6827;

P(158<=X<=190) = 0.9545;

P(150<=X & X>=190) = 0.0241;

P(150<=X & X>=198) = 0.0027;

P(X<=166) = 0.1587

5.На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от
математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

In [32]:
Z = (190 -178)/25**(1/2)
Z

2.4